.github/workflows/ci-tests.yml

```
@@ -0,0 +1,27 @@
+name: CI Tests
+
+on: [push]
```

I thin we want to modify this to mirror what we have in travis, pushes to master only and PR additions, 
- https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions#targeting-specific-branches

- https://github.community/t/trigger-workflow-for-a-pull-request-and-for-all-subsequent-commits-in-it/17996/2
```suggestion
on:
  pull_request:
  push:
    branches:    
      - master
```
current travis config https://github.com/zooniverse/front-end-monorepo/blob/3ac16a3d1f7132fc1af42e2f793e331a8cd2e106/.travis.yml#L6-L8